### PR TITLE
fix: init create dir before

### PR DESCRIPTION
**Pull request by typicode**

### CODE REVIEW

File: bin.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ if (a == 'init') {
   d.scripts.prepare = 'husky'
   w('package.json', JSON.stringify(d, null, /\t/.test() ? '\t' : 2) + '\n')
   process.stdout.write(i())
+	try { f.mkdirSync('.husky') } catch {}
```
Comment (glensc) [inline bin.js:14]: unidented commit. seems to use tab while rest of the code use spaces

Comment (typicode) [reply bin.js:14]: That was a mistake. It was fixed by another commit (using tabs with .editorconfig)

Comment (glensc) [reply bin.js:14]: Time to dogfood and setup git hooks to husky repo to auto lint? ;)

Comment (typicode) [reply bin.js:14]: Ahah probably :D Dog fooding json-server already.

**Pull request merged**